analyzere/extras/visualizations.py

```diff
@@ -0,0 +1,272 @@
+import hashlib
+
+from analyzere import LayerView
```

Comment:
I tried to import this module and it did not for me.  Python complained about not being able to find the class `LayerView`.  I did have the analyzere package installed but I believe that the analyzere module in this package was masking the original.  I suggest we change the directory structure here to `analyzere_extras/visualizations/`.